### ISSUE TITLE
FR: Display wifi logo on home page

### ISSUE THREAD

**Issue by Huahddd** [koreader/koreader#12411]
**Does your feature request involve difficulty completing a task? Please describe.**
It is inconvenient to check whether the wifi is turned on, the battery level, the time, etc. You need to click once to check the battery level and time.

**Describe the solution you'd like**
1. Add a WiFi icon to the location shown to confirm whether WiFi is turned on
2. Place WiFi, battery level, time, etc. on the home page for easy viewing

**Additional context**
![1](https://github.com/user-attachments/assets/7589a7e9-e329-4de1-808a-d85aba274065)

**Comment by poire-z** [comment]
We don't really have a "home page".
Most of us have all these info in the reader footer/status bar.
But I guess a Wifi status icon could be shown in that top menu footer (if it's cheap, which it must be if used in the footer).

**Comment by NiLuJe** [comment]
> if it's cheap

That depends on the platform ;).

**Comment by NiLuJe** [comment]
A couple of very personal notes about that:

1. I would hate it.
2. It's going to be problematic in terms of available width, we already have a couple of places (multiple pages) where we're veeeeery close to overflowing (and, on a Sage with two battery icons, we actually do, FWIW ;)).

**Comment by Huahddd** [comment]
koreader is a great software, but its current UI is too crude. Have you ever considered improving the UI of koreader? Maybe you can refer to the UI of Kindle and kobo.

**Comment by Commodore64user** [comment]
> > if it's cheap
> 
> That depends on the platform ;).

Even if it isn't, i think it would help save power in the long term. You would be made aware your wifi is on and would act on it instead of just inadvertently being on the entire time.

And yes, i am aware of that setting that kills wifi after a while but that doesn't really work on kindle very well.


> I would hate it.

That is a bit harsh. Don't be a hater, be cool 😎. In Taylor's own words, haters gonna hate.

**Comment by Frenzie** [comment]
> (Although I don't recall if we currently have any left where it isn't).

It was originally gated behind isAndroid (as being the only platform where it was sufficiently cheap back then).

**Comment by jonnyl2** [comment]
So would I noticeably save some battery life if I remove the Wifi status indicator from the status bar of a Kobo device?

**Comment by NiLuJe** [comment]
No, but you would by enabling "Disable Wi-Fi connection when inactive", which does work on Kobo ;).

(It might actually be the default, I can't recall).

**Comment by Frenzie** [comment]
> So would I noticeably save some battery life if I remove the Wifi status indicator from the status bar of a Kobo device?

Also note that we since implemented something similar on Kobo regardless, but the latency in question is related to waiting for a response while turning a page or opening the menu, which is just very unpleasant. Any impact on battery would be negligible.
